webapp/src/shared/components/Row/ScoreRow.css

```diff
@@ -5,35 +5,54 @@
   grid-template-rows: 1fr;
   width: 100%;
   min-width: fit-content;
-  padding: 0.5rem 1.75rem;
+  padding: 0.5rem 0.5rem;
   border-radius: 0.5rem;
-  column-gap: 1.25rem;
+  column-gap: 0.5rem;
   color: var(--color-light);
   background-color: var(--color-dark);
   border: 0;
   align-items: center;
   flex-direction: row;
 }
-.scorerow_text_wrapper .scorerow_text_upper_wrapper > *,
-.scorerow_text_wrapper .scorerow_text_lower_wrapper > * {
+.scorerow_text_upper_wrapper_p1 > *,
+.scorerow_text_upper_wrapper_score > *,
+.scorerow_text_upper_wrapper_p2 > *,
+.scorerow_text_lower_wrapper > * {
   margin: 0;
   overflow: hidden;
-  grid-column: span 1;
-  max-width: 70vw;
-  white-space: pre;
+  white-space: nowrap;
   text-overflow: ellipsis;
   padding: 0;
 }
 
 .scorerow_text_upper_wrapper {
-  display: flex;
-  justify-content: space-between;
+  display: grid;
+  grid-template-columns: minmax(0, 1fr) 4.2rem minmax(0, 1fr);
+  grid-template-rows: 1fr;
+  column-gap: 0.5rem;
+  width: 100%;
+  min-width: fit-content;
+  align-items: center;
+  flex-direction: row;
 }
+
 .scorerow_text_lower_wrapper {
   display: flex;
   justify-content: space-evenly;
```

Comment:
question: What do you think about using a grid here to align the lower text info?

Reply:
I will change it

Reply:
Done. I center the content in each grid area because I didn't like the empty space in the row if the mode is left-aligned ant the date on the right on the desktop version. But if you want to change it, please feel free